assets/RFC002-parse-response-body.puml

```diff
@@ -0,0 +1,18 @@
+@startuml
```

Comment:
I am using the plantuml proxy service for this. It is documented here: https://blog.anoff.io/2018-07-31-diagrams-with-plantuml/#github-integration

The rendered version is embedded in the RFC once it is merged and available on master.

I purposely did not include a rendered version so that it cannot get outdated.

Reply:
> The rendered version is embedded in the RFC once it is merged and available on master.

Shame we cannot review the rendered version

Reply:
You are free to just copy/paste it into plantuml online: http://plantuml.com/